stoobly_agent/record.py

```diff
@@ -255,9 +256,19 @@ def __upload_request(flow, api, settings):
     if Settings.instance().is_debug():
         # Build file path, replace slashes with underscores
         request_path = request.path.replace('/', '_')
-        file_path = os.path.join(tempfile.gettempdir(), request_path, str(int(time.time() * 1000)))
+        timestamp = str(int(time.time() * 1000))
+        file_path = os.path.join(tempfile.gettempdir(), 'stoobly', request_path, timestamp)
```

Comment:
Maybe make 'stoobly' a constant?

Reply:
Done in https://github.com/Stoobly/stoobly-agent/pull/2/commits/8c7b22900f49a8796c0b590148fa5f534c8d4f55